### PR TITLE
refine subchangfeedinfo clean strategy

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 

Currently we don't have `SubChangefeedInfo` clean strategy.

### What is changed and how it works?

- ~When a capture closes normally, clean all `SubChangefeedInfo` belongs to it. We can't do this, we must at least record the replication checkpoint information, just reuse the subchangefeedinfo~
- When the owner marks a processor down, clean both `CaptureInfo` and all `SubChangefeedInfo` belongs to the capture. Table rebalance strategy will be added in another PR

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: cdc/capture.go [hunk 1]
```diff
@@ -97,7 +98,7 @@ func (c *Capture) OnRunProcessor(p *processor) {
 // OnStopProcessor implements processorCallback.
 func (c *Capture) OnStopProcessor(p *processor, err error) {
 	// TODO: handle processor error
-	log.Info("stop to run processor", zap.String("changefeed id", p.changefeedID), zap.Error(err))
+	log.Info("stop to run processor", zap.String("changefeed id", p.changefeedID), util.ZapErrorFilter(err, context.Canceled))
```
Comment (suzaku) [inline cdc/capture.go]: How do we decide when to use `util.ZapErrorFilter(err, context.Canceled)` or not?

Comment (amyangfei) [reply cdc/capture.go]: In most cases we just ignore the `context.Canceled` error

**Comment by amyangfei** [comment]
have a better solution in #214

**Pull request closed without merging**